tests/support/pytest/transport.py

```diff
@@ -223,17 +253,24 @@ def __init__(self, master_config, minion_config, **collector_kwargs):
             self.master_config["interface"],
             self.master_config["publish_port"],
             self.aes_key,
-            **self.collector_kwargs
+            **self.collector_kwargs,
         )
 
     def run(self):
+        import queue
```

Comment:
```suggestion
        import queue
```
Why not top level import?